src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1950,7 +1950,7 @@ private void OpenRecent(object path)
 
         private bool CanOpenRecent(object path)
         {
-            return HomeSpace.RunSettings.RunEnabled;
+            return HomeSpace.RunSettings.RunEnabled && !HomeSpace.GraphRunInProgress;
```

Comment:
Since these two conditions are used in several places, how about wrapping them in a new property that is appropriately named to convey its exact meaning?

Reply:
done